packages/suite-web/e2e/tests/wallet/send-form-regtest.test.ts

```diff
@@ -46,7 +46,7 @@ describe('Send form for bitcoin', () => {
         // add locktime
         cy.getTestElement('add-locktime-button').click();
 
-        cy.getTestElement('locktime-input').type('100');
+        cy.getTestElement('locktime-input').type('1000');
```

Comment:
isn't the problem that there isn't any meaningful error shown if locktime is lower then current block?

Reply:
I am aware of that. Just the fact that we needed to discuss it makes me question whether UX of this feature is good enough for eddies to understand why they can not submit the send form